t/validate-recursive.t

```diff
@@ -0,0 +1,53 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator 'validate_json';
+
+{
+  use Mojolicious::Lite;
+  post '/' => sub {
+    my $c = shift;
+    my @errors = validate_json $c->req->json, 'data://main/spec.json';
+    $c->render(status => @errors ? 400 : 200, text => "@errors");
```

Comment:
Isn't it better to do ```json => \@errors``` ? I believe that would contribute to better unit testing than content_like().